### PR TITLE
Galvo projector: adds ability to run polygons to Utilities::DAGalvo device.

### PR DESCRIPTION
**Pull request by nicost**
There is ample ambiguity who controls the shutter and channel config while photo-targetting, the UI code or the device adapter.

We should discuss the best path forward, but in the mean time this works.

### CODE REVIEW

File: DeviceAdapters/Utilities/DAGalvo.cpp [hunk 1]
```diff
@@ -262,8 +283,14 @@ int DAGalvo::SetPolygonRepetitions(int repetitions)
 
 int DAGalvo::RunPolygons()
 {
-   return DEVICE_NOT_YET_IMPLEMENTED;
+   for (int i = 0; i < polygons_->size(); i++) {
+      DAPolygon* polygon = polygons_->at(i);
+      std::pair<double, double> xyPair = polygon->getVertex(0);
```
Comment (marktsuchida) [inline DeviceAdapters/Utilities/DAGalvo.cpp:288]: Did you mean `getVertex(i)`?

Comment (marktsuchida) [reply DeviceAdapters/Utilities/DAGalvo.cpp:288]: Never mind, I guess this is intentional(?)

**Comment by marktsuchida** [comment]
What is the reason for disabling the `SleepMs(1)` in `SnapImage()` on Windows?

**Comment by nicost** [comment]
Yikes, that is a left over from trying to get those sleeps to sleep shorter.  I'll add another PR to bring this back.

File: DeviceAdapters/Utilities/DAGalvo.cpp (hunk 1 above)
Comment (nicost) [reply DeviceAdapters/Utilities/DAGalvo.cpp:288]: Yup, so far it only targets points, simplified by only using the first entry of the polygon.  Could be extended in the future to draw the complete polygon.

Sorry for merging so fast!  I could have avoided some problems by waiting.

**Pull request merged**